### PR TITLE
add -e to echo command

### PR DESCRIPTION
**Pull request by thredreams**
To solve the problem of invalid escape characters, add the - e option to the echo command in the ". mk" files in the gateway/tool/make directory

### CODE REVIEW

**Comment by Xunzhuo** [comment]
Plz sign your commit first, and describe the details of the issue you pointed out.

File: tools/make/common.mk [hunk 1]
```diff
@@ -112,6 +112,6 @@ export USAGE_OPTIONS
 .PHONY: help
 help:
 	@echo "Envoy Gateway is an open source project for managing Envoy Proxy as a standalone or Kubernetes-based application gateway\n"
```
Comment (Xunzhuo) [inline tools/make/common.mk:114]: Add -e to here too.

**Review by Xunzhuo — CHANGES_REQUESTED** [review]
You need to add `SHELL=/bin/bash` to common.mk, to specify the shell which makefile is using.

**Comment by thredreams** [comment]
> You need to add `SHELL=/bin/bash` to common.mk, to specify the shell which makefile is using.
> 
> Others LGTM. Thank you @thredreams.

In fact, I didn't do this at the beginning because I was worried about whether there were some commands in the file that should not be executed in bash

**Comment by Xunzhuo** [comment]
> In fact, I didn't do this at the beginning because I was worried about whether there were some commands in the file that should not be executed in bash

Not for now : )

**Comment by Xunzhuo** [comment]
Thanks @thredreams, after CI passed, I am going to merge this.

**Pull request merged**